sources/packages/web/src/components/students/ApplicationDetails.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <v-container>
+    <v-card>
+      <formio
+        formName="studentapplicationdetails"
+        :data="initialData"
+        @customEvent="customEventCallback"
+      ></formio>
+    </v-card>
+    <v-card>
+      <v-container>
+        <p class="category-header-large">Track your application</p>
+        <div class="progress">
```

Comment:
IMO I would strongly recommend to make progress bar functionality as a individual component and plugin the component to application details component.

Reply:
@dheepak-aot can you explain why you want the progress bar as a separate component?